packages/server/package.json

```diff
@@ -81,19 +81,19 @@
   },
   "homepage": "https://github.com/apollographql/apollo-server#readme",
   "engines": {
-    "node": ">=14.16.0"
+    "node": ">=16.14.0"
   },
   "dependencies": {
     "@apollo/cache-control-types": "^1.0.2",
     "@apollo/server-gateway-interface": "^1.1.0",
     "@apollo/usage-reporting-protobuf": "^4.1.0",
-    "@apollo/utils.createhash": "^2.0.0",
-    "@apollo/utils.fetcher": "^2.0.0",
-    "@apollo/utils.isnodelike": "^2.0.0",
-    "@apollo/utils.keyvaluecache": "^2.1.0",
-    "@apollo/utils.logger": "^2.0.0",
-    "@apollo/utils.usagereporting": "^2.0.0",
-    "@apollo/utils.withrequired": "^2.0.0",
+    "@apollo/utils.createhash": "^3.0.0",
+    "@apollo/utils.fetcher": "^3.0.0",
+    "@apollo/utils.isnodelike": "^3.0.0",
+    "@apollo/utils.keyvaluecache": "^3.0.0",
+    "@apollo/utils.logger": "^3.0.0",
+    "@apollo/utils.usagereporting": "^3.0.0",
+    "@apollo/utils.withrequired": "^3.0.0",
     "@graphql-tools/schema": "^9.0.0",
```

Comment:
We should be able to bump this to ^10.0.0 now, too

Reply:
Thanks, yeah a couple major dep updates have stacked up since I started this PR that I'll get merged once I get back to it 👍